pyam/iiasa.py

```diff
@@ -66,14 +66,49 @@ def auth(self):
     @lru_cache()
     def metadata(self):
         """
-        Metadata (e.g., models, scenarios, etc.) in the connected data
+        Metadata (e.g., models, scenarios, etc.) of the connected data
         source
         """
         url = self.base_url + 'runs?getOnlyDefaultRuns=false'
         headers = {'Authorization': 'Bearer {}'.format(self.auth())}
         r = requests.get(url, headers=headers)
         return pd.read_json(r.content, orient='records')
 
+    @lru_cache()
+    def available_scenario_metadata(self):
+        """
+        List all scenario metadata available in the connected data source
+        """
+        url = self.base_url + 'metadata/types'
+        headers = {'Authorization': 'Bearer {}'.format(self.auth())}
+        r = requests.get(url, headers=headers)
+        return pd.read_json(r.content, orient='records')['name']
+
+    @lru_cache()
+    def scenario_metadata(self):
```

Comment:
I would suggest to add the kwarg `default`(default `True`), see https://github.com/iiasa/ixmp/blob/88edf76fa12e82df895b5894916e9570cb31cd76/ixmp/core.py#L161 for the equivalent treatment in `ixmp`